comp/api/api/utils/stream/stream.go

```diff
@@ -92,3 +105,41 @@ func GetStreamFunc(messageReceiverFunc func() MessageReceiver, streamType, agent
 		}
 	}
 }
+
+// ExportStreamLogs export output of stream-logs to a file
+func ExportStreamLogs(config config.Component, streamLogParams *LogParams) error {
+	ipcAddress, err := pkgconfig.GetIPCAddress()
```

Comment:
Is this the best place for this functionality? All other usages of `GetIPCAddress` are in entry points, such as `cmd/.../subcommands/...` or `pkg/.../subcommands/...`, whereas this is within a utility package of the API component. Putting an IPC into a subcommand entry point makes sense because a command-line invocation of the agent can contact a running agent server process, but here it seems like the code may already be running in an agent server? I'm not convinced this makes sense from a code organization standpoint.